Updated Kapacitor Kafka event handler partitioning options (#2697)
influxdata/docs-v2: content/kapacitor/v1.6/event_handlers/kafka.md

  insecure-skip-verify = false
```

{{% note %}}

> Multiple Kafka clients may be configured by repeating `[[kafka]]` sections.

Reviewer: ```suggestion
Multiple Kafka clients may be configured with multiple `[[kafka]]` sections in TOML.
```

The `id` acts as a unique identifier for each configured Kafka client.
{{% /note %}}

#### enabled
Set to `true` to enable the Kafka event handler.

#### id
A unique identifier for the Kafka cluster.

#### brokers
List of Kafka broker addresses using the `host:port` format.

#### timeout
Timeout on network operations with the Kafka brokers.
If set to 0, a default of 10s is used.

#### batch-size
The number of messages batched before being sent to Kafka.
If set to 0, a default of 100 is used.

#### batch-timeout
The maximum amount of time to wait before flushing an incomplete batch.
If set to 0, a default of 1s is used.

#### use-ssl
Enable SSL communication.
Must be `true` for other SSL options to take effect.

#### ssl-ca
Path to certificate authority file.

#### ssl-cert
Path to host certificate file.

#### ssl-key
Path to certificate private key file.

#### insecure-skip-verify

> Use SSL but skip chain and host verification
> _(required if using a self-signed certificate)_.

Reviewer: ```suggestion
Use SSL but skip chain and host verification.
_(Required if using a self-signed certificate.)_
```
